es/resource_elasticsearch_index_test.go

```diff
@@ -273,6 +273,19 @@ resource "elasticsearch_index" "test" {
 
   depends_on = [elasticsearch_index_template.test]
 }
+`
+
+	testAccElasticsearchIndexWithSimilarityConfig = `
+resource "elasticsearch_index" "test_similarity_config" {
+  name = "terraform-test-update-similarity-module"
+  number_of_shards   = 1
+  number_of_replicas = 1
+  index_similarity_default = jsonencode({
+    type: "BM25",
+    b: 0.25,
+    k1: 1.2
```

Comment:
The linter complained about HCL formatting, I think you're missing a trailing comma here

Reply:
@phillbaker ran linter locally and updated :)